src/xcode/ENA/ENA/Source/Scenes/ExposureDetection/Info/HygieneRules/HygieneRulesInfoViewController.swift

```diff
@@ -48,7 +48,7 @@ class HygieneRulesInfoViewController: DynamicTableViewController {
 		tableView.separatorStyle = .none
 		tableView.contentInsetAdjustmentBehavior = .never
 		
-		navigationItem.rightBarButtonItem = CloseBarButtonItem(onTap: dismiss)
+		navigationItem.rightBarButtonItem = CloseBarButtonItem(onTap: dismiss, accessibilityIdentifierSuffix: "HygieneRules")
```

Comment:
Would be nicer to have the strings as a static let in the AccessibilityIdentifiers enum. This way a refactoring cleaning would be supported by the compiler.

Reply:
done